client/src/telethon/_impl/client/events/queries.py

```diff
@@ -75,8 +88,10 @@ async def get_message(self) -> Optional[Message]:
         """
         Get the :class:`~telethon.types.Message` containing the button that was clicked.
 
-        If the message is too old and is no longer accessible, :data:`None` is returned instead.
+        If the message is inline, or too old and is no longer accessible, :data:`None` is returned instead.
```

Comment:
But isn't the purpose of using `InputMessageCallbackQuery` the fact that it can work? `functions.messages.get_messages` doesn't need a chat. `CherryPickedList` does require one, so I see the problem.

But we should be able to fetch the message without a chat. So maybe we need to rethink this method's implementation.